tests/page-objects/common/common.po.js

```diff
@@ -151,7 +151,9 @@ module.exports = {
   confirmDelete: async () => {
     await helper.waitUntilReady(deleteButton);
     await deleteButton.click();
-  }
+  },
+
+  openSubmenu: openSubmenu,
```

Comment:
Non-blocking, but the object syntax can be shortened to this:
```suggestion
  openSubmenu,
```

Reply:
I use this longer syntax when the file uses longer exports. 
Visually, for me, when there's method definitions after method definition and then a random word, it kinda throws me off :) 

When exports are short (visually short),I like the one word short syntax.